### DISCUSSION TITLE
In the recognition training, though the precision of the training set reaches 90, the precision of the validation set is still 70. How to handle this case?

### DISCUSSION THREAD

**Discussion by livingbody** [PaddlePaddle/PaddleOCR#6017] [Q&A] [answered]
In the recognition training, though the precision of the training set reaches 90, the precision of the validation set is still 70. How to handle this case?

**Comment by Evezerest** [comment, marked as answer]
This may be due to overfitting. There are two solutions you can try: (1) Introduce more augmentation methods or increase the probability of augmentation, and the default value is 0.4; (2) increase the 12 decay value of the system.
